news/_posts/releases/2019-06-25-prestashop-1-7-6-0-rc-release.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "Release note Release Candidate 1.7.6"
+subtitle: "A few weeks after the beta, here it is: the first release candidate for PrestaShop 1.7.6.0 is now ready for you to test!"
+date:   2019-06-25 16:00:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - rc
+ - minor
+ - releases
+ - 1.7.6.x
+---
+
+
+This is the last step before releasing the final 1.7.6.0. It aims at making sure that all regressions found during the beta have been fixed, and that no new ones appeared. If no new regression is reported by June 27, we will make the final version of 1.7.6.0 available.
```

Comment:
If no new regression is reported by July 1st, we will make the final version of 1.7.6.0 available.

Reply:
done